### PR TITLE
Design system form field input

### PR DESCRIPTION
**Pull request by bhushankhope**

### CODE REVIEW

**Review by LibbyUX — DISMISSED** [review]
Excellent! Thank you @bhushankhope !

File: libs/design-system/input/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export * from './lib/input-styles/input-styles.component';
```
Comment (axdanbol) [inline libs/design-system/input/src/index.ts:1]: No need to export this component

File: libs/design-system/input/src/lib/input-styles/input-styles.component.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
+.mat-mdc-form-field {
+  font-family: 'Metropolis';
```
Comment (axdanbol) [inline libs/design-system/input/src/lib/input-styles/input-styles.component.scss:2]: Read from a --sys variable. Add `-font` to get just the font family from the variables, i.e. `--sys-label-small-font`

File: libs/design-system/input/src/lib/input.component.stories.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    }),
+  ],
+};
+export default meta;
+type Story = StoryObj;
+
+export const Primary: Story = {
+  args: {},
+  render: () => ({
+    template: `
```
Comment (axdanbol) [inline libs/design-system/input/src/lib/input.component.stories.ts:25]: Please split into 4 different stories

**Review by axdanbol — CHANGES_REQUESTED** [review]

**Review by axdanbol — DISMISSED** [review]

**Pull request merged**